src/pages/LabIntakeInput.js

```diff
@@ -21,9 +153,14 @@ const LabIntakeInput = ({ labState, setLabState }) => {
                   type="number"
                   inputMode="decimal"
                   name="crp"
-                  onChange={handleInputChange}
+                  onChange={handleCrpChange}
                   value={labState["crp"]}
                 />
+                {crpError ? (
+                  <p className="lab-error-message">
+                    upto one decimal point only
+                  </p>
+                ) : null}
```

Comment:
Please update all inputs similarly:
```suggestion
                {labErrors.crp && (
                  <p className="lab-error-message">
                    upto one decimal point only
                  </p>
                )}
```